packages/rrweb/test/replayer.test.ts

```diff
@@ -783,6 +786,125 @@ describe('replayer', function () {
     await assertDomSnapshot(page);
   });
 
+  it('has a pointer for touch interactions with pointerIds', async () => {
+    await page.evaluate(`events = ${JSON.stringify(touchAllPointerEvents)}`);
+    await page.evaluate(`
+      const { Replayer } = rrweb;
+      const replayer = new Replayer(events);
+      replayer.play();
+    `);
+
+    // No active pointers should exist
+    await expect(
+      await page.evaluate(
+        () => document.querySelectorAll('.touch-active')!.length,
+      ),
+    ).toEqual(0);
+
+    await page.evaluate(`
+      replayer.pause(101);
+    `);
+
+    // 2 pointers should exist
+    await expect(
+      await page.evaluate(
+        () => document.querySelectorAll('.touch-active')!.length,
+      ),
+    ).toEqual(2);
+
+    await page.evaluate(`
+      replayer.play();
+    `);
+
+    await page.waitForTimeout(160);
+
+    // Both pointers should be removed after the TouchEnd event
+    await expect(
+      await page.evaluate(
+        () => document.querySelectorAll('.replayer-mouse')!.length,
+      ),
+    ).toEqual(0);
+  });
+
+  // This should't happen, but we want to test/capture the behavior
+  it('has a pointer for touch interactions with some pointerIds', async () => {
+    await page.evaluate(`events = ${JSON.stringify(touchSomePointerEvents)}`);
+    await page.evaluate(`
+      const { Replayer } = rrweb;
+      const replayer = new Replayer(events);
+      replayer.play();
+    `);
+
+    // No active pointers should exist
+    await expect(
+      await page.evaluate(
+        () => document.querySelectorAll('.touch-active')!.length,
+      ),
+    ).toEqual(0);
+
+    await page.evaluate(`
+      replayer.pause(101);
+    `);
+
+    // 2 pointers should exist
+    await expect(
+      await page.evaluate(
+        () => document.querySelectorAll('.touch-active')!.length,
+      ),
+    ).toEqual(2);
+
+    await page.evaluate(`
+      replayer.play();
+    `);
+
+    await page.waitForTimeout(220);
```

Comment:
for some reason this test doesn't work with just `replayer.pause(220)`, had to add a `waitForTimeout` instead